<ui version="4.0" >
 <class>cs8WagoNodeWidgetBase</class>
 <widget class="QWidget" name="cs8WagoNodeWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>&amp;Network Settings</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Wago Node IP Address</string>
          </property>
          <property name="buddy" >
           <cstring>leAddress</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&amp;St√§ubli IP Address</string>
          </property>
          <property name="buddy" >
           <cstring>leCtrlAddress</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Subnet &amp;Mask</string>
          </property>
          <property name="buddy" >
           <cstring>leSubnet</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>&amp;Gateway</string>
          </property>
          <property name="buddy" >
           <cstring>leGateway</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLineEdit" name="leAddress" >
          <property name="inputMask" >
           <string>009.009.009.009; </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leCtrlAddress" >
          <property name="inputMask" >
           <string>00D.00D.00D.00D; </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leSubnet" >
          <property name="inputMask" >
           <string>000.000.000.000; </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leGateway" >
          <property name="inputMask" >
           <string>000.000.000.000; </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>IO Settings</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Read Word Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Write Word Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Read Bit Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>Write Bit Address</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QSpinBox" name="leReadWordAddress" >
          <property name="minimumSize" >
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="leWriteWordAddress" >
          <property name="minimumSize" >
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="leReadBitAddress" >
          <property name="minimumSize" >
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="leWriteBitAddress" >
          <property name="minimumSize" >
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="buttonSymbols" >
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>General Settings</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>Timeout (ms)</string>
        </property>
        <property name="buddy" >
         <cstring>leTimeout</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leTimeout" />
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" colspan="2" >
       <widget class="QLabel" name="label_11" >
        <property name="text" >
         <string>Configuration Byte</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QLineEdit" name="leConfigurationByte" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Scan rate (ms)</string>
        </property>
        <property name="buddy" >
         <cstring>leScanRate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leScanRate" />
      </item>
      <item row="1" column="2" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Protocol</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QComboBox" name="cbProtocol" >
        <item>
         <property name="text" >
          <string>Modbus / TCP</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Modbus / UDP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>301</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4" colspan="2" >
       <widget class="QCheckBox" name="cbBootPRequest" >
        <property name="text" >
         <string>BootP Request enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
